include/dxc/WinAdapter.h

```diff
@@ -912,16 +912,31 @@ unsigned int SysStringLen(const BSTR bstrString);
 #define CP_ACP 0
 #define CP_UTF8 65001 // UTF-8 translation.
 
-// Convert Windows codepage value to locale string
-const char *CPToLocale(uint32_t CodePage);
+// Sets a locale for the specified Windows codepage
+const char *SetLocaleForCodePage(int Category, uint32_t CodePage);
```

Comment:
Maybe SetLocaleForCodePage could be moved inside ScopedLocale, further encapsulating all this stuff.

Reply:
I can do that.  I remove all places where callers were manually setting the locale.  They should all be using the ScopedLocale class now.